lib/cartopy/io/img_tiles.py

```diff
@@ -497,6 +497,50 @@ def find_images(self, target_domain, target_z, start_tile=None):
                 yield self.tms_to_quadkey(tile, google=True)
 
 
+class OrdnanceSurvey(GoogleWTS):
+    # https://developer.ordnancesurvey.co.uk/os-api-framework-agreement
+    # https://apidocs.os.uk/docs/os-maps-wmts
+
+    def __init__(self, apikey, layer='Road', style=True,
+                 desired_tile_form='RGB'):
+        """
+        Parameters
+        ----------
+        apikey: required
+            The authentication key provided by OS to query the maps API
+        layer: optional
+            The style of the OS map tiles. One of 'Outdoor', 'Road',
+            'Light', 'Night', 'Leisure'. Defaults to 'Road'.
+        style: optional
+            Indicates if the map should be styled. Defaults to True.
+        desired_tile_form: optional
+            Defaults to 'RGB'.
```

Comment:
As with `layer` above, could we have an indication of valid options here (and perhaps a description of what it will change for OS tiles).

Reply:
I agree with @dkillick oln this one; it would be much more user-friendly with a set of possible options in here and a more complete description.

Reply:
The style parameter no longer exists in the current version of this PR.